docs/tutorials/gluon/transforms.md

```diff
@@ -0,0 +1,156 @@
+
+* **Step 1** is to calculate the mean and standard deviation of each channel for the entire training dataset.
+* **Step 2** is to use these statistics to normalize each sample for training and for inference too.
+
+When using pre-trained models, you need to use the same normalization statistics that were used for training. Models from the Gluon Model Zoo expect normalization statistics of `mean=[0.485, 0.456, 0.406]` and `std=[0.229, 0.224, 0.225]`.
+
+When training your own model from scratch, you can estimate the normalization statistics from the training dataset (not test dataset) using the code snippet found below. Models often benefit from input normalization because it prevents saturation of activations and prevents certain features from dominating due to differences in scale.
+
+
+```python
+# estimate channel mean and std
+sample_means = np.empty(shape=(len(dataset), 3))
+sample_stds = np.empty(shape=(len(dataset), 3))
+for idx in range(len(dataset)):
+    sample_data = dataset[idx][0].asnumpy()
+    sample_means[idx] = sample_data.mean(axis=(1,2))
+    sample_stds[idx] = sample_data.std(axis=(1,2))
+print("channel means: {}".format(sample_means.mean(axis=0)))
+print("channel stds: {}".format(sample_stds.mean(axis=0)))
+```
+
+We can create our `Normalize` transform using these statistics and apply it to our `dataset` as before.
+
+
+```python
+normalize_fn = mx.gluon.data.vision.transforms.Normalize(mean=[0.49139969, 0.48215842, 0.44653093],
+                                                         std=[0.20220212, 0.19931542, 0.20086347])
+dataset = dataset.transform_first(normalize_fn)
+```
+
+And now when we inspect the values for a single sample, we find that it's no longer bounded by 0 and 1.
+
+
+```python
+sample_data, sample_label = dataset[sample_idx]
+print("data range: {} to {}".format(sample_data.min().asscalar(),
+                                    sample_data.max().asscalar()))
+```
+
+### `Compose`
+
+We've now seen two examples of transforms: [`ToTensor`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=totens#mxnet.gluon.data.vision.transforms.ToTensor and `Normalize`. We applied both transforms to our `dataset` through repeated calls to `transform_first`, but Gluon has a dedicated transform to stack other transforms that's preferred. With [`Compose`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=compose#mxnet.gluon.data.vision.transforms.Compose) we can choose and order the transforms we want to apply.
+
+Caution: ordering of transforms is important. e.g. [`ToTensor`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=totens#mxnet.gluon.data.vision.transforms.ToTensor) should be applied before `Normalize`, but after `Resize` and `CenterCrop`.
```

Comment:
This is very important information! Are there more transforms where the order matters?

Reply:
Will add a note to augmentation tutorial about this too.